pages/components/LinkedButton.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { ComponentProps, MouseEventHandler, ReactPropTypes } from "react";
```

Comment:
```suggestion
import { MouseEventHandler } from "react";
```
remove unused imports